packages/google-vertex/src/google-vertex-image-model.ts

```diff
@@ -96,5 +112,5 @@ export class GoogleVertexImageModel implements ImageModelV1 {
 // minimal version of the schema, focussed on what is needed for the implementation
 // this approach limits breakages when the API changes and increases efficiency
 const vertexImageResponseSchema = z.object({
-  predictions: z.array(z.object({ bytesBase64Encoded: z.string() })),
+  predictions: z.array(z.object({ bytesBase64Encoded: z.string() })).optional(),
```

Comment:
prefer `nullish` over `optional`. I have seen API change and suddenly return `null` instead of nothing, causing breakages with `optional`. `nullish` covers both.